deliveries/Archisinal-milestone_1.md

```diff
@@ -0,0 +1,41 @@
+
+* **Application Document:** [Archisinal](https://github.com/use-inkubator/Ecosystem-Grants/blob/master/applications/Archisinal.md)
+
+**Context** (optional)
+> The 1st milestone focuses on the Smart Contracts part of our Artisanal MVP. It includes basic marketplace features like administering buyers and creators, selling NFTs with fixed prices or timed auctions, managing collections, etc.
+
+**Deliverables**
+
+| Number | Deliverable | Link                                                                                   | Notes                                                                                                                                                                                        |                             
+| ------------- | ------------- |----------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 0a. | License | [LICENSE](https://github.com/archisinal/marketplace-contracts/blob/main/LICENSE)                   | MIT                                                                                                                                                                                          |
+| 0b. | Documentation | [README.md](https://github.com/archisinal/marketplace-contracts/blob/main/README.md)                      | Entry-point of docs is README.md, then use `cargo doc --open` to check all docs.                                                                                                             |
+| 0c. | Testing and Testing Guide | [TESTING.md](https://github.com/archisinal/marketplace-contracts/blob/main/TESTING.md)             | Created E2E, Security, Performance tests, and testing guide for them.                                                                                                                        |
+| 0d. | Docker | [Dockerfile](https://github.com/archisinal/marketplace-contracts/blob/main/Dockerfile), [CI](https://github.com/Archisinal/marketplace-contracts/blob/main/.github/workflows/ci.yml)             | Created dockerfile for testing, also CI for automatic checks.                                                                                                                                |
+| 0e. | Article | [Medium: Archisinal Technologies — Genesis Scoping & MVP](https://medium.com/p/3df888e1de98)                                                                            |                                                                                                                                                                                              |
```

Comment:
nitpick: I'd like to see more pictures! Architecture can be beautiful, leverage that instead of having walls of text

Reply:
Sure) We’ll work on the medium article and improve visuals. 

Also, we updated the [README.md](https://github.com/Archisinal/marketplace-contracts/blob/main/README.md) with UML diagram, to make our contacts architecture more readable.

Reply:
The diagram is a nice touch, but it's got too much detail imo. Having function names in there means it's probably going to become outdated quickly. This is up to you at this point though.